src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -477,7 +477,7 @@ export default {
         const filing = this.filings[i].filing
         if (filing?.header?.date) {
           let filingDate = filing.header.date.slice(0, 10)
-          if (filing.header.availableInColinOnly) {
+          if (filing.header.availableInColinOnly || filing.header.inColinOnly) {
```

Comment:
I'm not sure when this was changed but now the UI handles both cases.

@thorwolpert COLIN filings go away in the new ledge, right?